Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
new file mode 100644
--- /dev/null	(date 1683539989114)
+++ b/.gitignore	(date 1683539989114)
@@ -0,0 +1,1 @@
+/InitialProject/lightning_logs/version_324/checkpoints/epoch=99-step=5200.ckpt
